RecoTracker/MkFit/plugins/MkFitHitConverter.cc

```diff
@@ -118,6 +127,27 @@ void MkFitHitConverter::produce(edm::StreamID iID, edm::Event& iEvent, const edm
   MkFitHitWrapper hitWrapper{mkFitGeom.trackerInfo()};
   mkfit::StdSeq::Cmssw_LoadHits_Begin(hitWrapper.eventOfHits(), {&hitWrapper.pixelHits(), &hitWrapper.outerHits()});
 
+  std::vector<mkfit::DeadVec> deadvectors(mkFitGeom.layerNumberConverter().nLayers());
+  const auto& siStripQuality = iSetup.getData(qualityToken_);
+  const auto& trackerGeom = iSetup.getData(geomToken_);
+  const auto& badStrips = siStripQuality.getBadComponentList();
+  for (const auto& bs : badStrips) {
+    const auto& surf = trackerGeom.idToDet(DetId(bs.detid))->surface();
+    const DetId detid(bs.detid);
+    const auto subdet = detid.subdetId();
+    const auto layer = ttopo.layer(detid);
+    const auto isStereo = ttopo.isStereo(detid);
+    bool isBarrel = (ttopo.side(detid) == static_cast<unsigned>(TrackerDetSide::Barrel));
+    bool isPlusSide = (ttopo.side(detid) == static_cast<unsigned>(TrackerDetSide::PosEndcap));
+    const auto ilay = mkFitGeom.layerNumberConverter().convertLayerNumber(subdet, layer, false, isStereo, isPlusSide);
+    //dump content of deadmodules.h in standalone setup
+    // std::cout << "deadvectors["<<ilay<<"].push_back({"<<surf.phiSpan().first<<","<<surf.phiSpan().second<<","
+    // 		<<(isBarrel ? surf.zSpan().first : surf.rSpan().first)<<","<<(isBarrel ? surf.zSpan().second : surf.rSpan().second)<<"});"<<std::endl;
+    deadvectors[ilay].push_back({surf.phiSpan().first,surf.phiSpan().second,
+	  (isBarrel ? surf.zSpan().first : surf.rSpan().first),(isBarrel ? surf.zSpan().second : surf.rSpan().second)});
+  }
+  mkfit::StdSeq::LoadDeads(hitWrapper.eventOfHits(), deadvectors);
```

Comment:
I suppose this code would go in `MkFitEventOfHitsProducer` in [trackreco/cmssw:mkFitIntegration](https://github.com/trackreco/cmssw/tree/mkFitIntegration).

Reply:
to be honest, I did not realize the branch I was using diverged from the main PR to central CMSSW. I am open to discuss what the best way to proceed is.

Reply:
I don't actually have strong opinions between this PR or #20 going in first. The changes are simple-enough to fix in #20.

Reply:
Following the merge of #20 this PR should be rebased (or re-worked in `MkFitEventOfHitsProducer`.

Reply:
I can re-work it MkFitEventOfHitsProducer. What is the correct branch to use for development and for the PR?

Reply:
For now it is still `CMSSW_11_2_0_mkFit_X`. It contains (via #20) all the modifications done in the PR-to-`cmssw`. We can discuss on Friday whether to continue the CMSSW side development on 11_2_0 or move to 12_0_0_pre3 (latter would simplify little bit future PRs to `cmssw`, but can have implications to input data files).

Reply:
I have reworked the changes in MkFitEventOfHitsProducer:
https://github.com/trackreco/cmssw/compare/CMSSW_11_2_0_mkFit_X...cerati:dead-modules-MkFitEventOfHitsProducer
I can close this PR and open a new one, please let me know. (However, looks like `RecoTracker/MkFit/test/reco_cfg.py` is currently broken, I am checking what is going on -- it complains about "No module named mkFitHitConverterDefault_cfi" maybe this is related to #20?)

Reply:
(was mkFitHitConverterDefault_cfi automatically created by RecoTracker/MkFit/plugins/MkFitHitConverter.cc? which has been removed in #20?)

Reply:
Certainly related to #20, I'll fix that, thanks. You can either force-push into your current `dead-modules` branch or create a new PR, whichever works.

Reply:
> Certainly related to #20, I'll fix that, thanks

Fixed in #25 (merged already).

Reply:
closing this PR, see #26